res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -46,14 +138,219 @@ DDJ200.seek = function(channel, control, value, status, group) {
     // of time given a constant turning angle.
     var duration = engine.getValue(group, "duration");
     var newPos = Math.max(0, oldPos + ((value - 64) * 0.2 / duration));
-    engine.setValue(group, "playposition", newPos); // Strip search
+
+    var deckNo = script.deckFromGroup(group);
+    var vgroup = "[Channel" + DDJ200.vDeckNo[deckNo] +"]";
+    engine.setValue(vgroup, "playposition", newPos); // Strip search
 };
 
 DDJ200.headmix = function(channel, control, value) {
-    if (value) {
+    // toggle headMix knob between -1 to 1
+    if (value) { // do nothing if button is released, i.e. value === 0
         var masterMixEnabled = (engine.getValue("[Master]", "headMix") > 0);
-        // toggle headMix knob between values of -1 and 1
         engine.setValue("[Master]", "headMix", masterMixEnabled ? -1 : 1);
         midi.sendShortMsg(0x96, 0x63, masterMixEnabled ? 0x7F : 0); // set LED
     }
 };
+
+DDJ200.toggleFourDeckMode = function(channel, control, value) {
+    if (value) { // only if button pressed, not releases, i.e. value === 0
```

Comment:
Everytime you only want to wrap the whole function body in `if (value) { ... }`, you could just early exit to reduce the nesting depth.

```javascript
if (!value) {
    return;
}
```

Reply:
I know.